capstone/target/appengine-staging/__static__/day_map.js

```diff
@@ -0,0 +1,27 @@
+let day_map_style =     [{
```

Comment:
The entire target directory can probably be in .gitignore. These files are built at runtime. We only want to review source code, not built code.

You would probably add something like capstone/target/* to .gitignore

If you can still see files in the target directory on Github, you did it wrong.